snuba/cli/migrations.py

```diff
@@ -128,6 +128,76 @@ def migrate(
     click.echo("Finished running migrations")
 
 
+@migrations.command()
+@click.option("-g", "--group", default=None)
+@click.option(
+    "-r",
+    "--readiness-state",
+    multiple=True,
+    type=click.Choice([r.value for r in ReadinessState], case_sensitive=False),
+    default=None,
```

Comment:
According to [this](https://click.palletsprojects.com/en/stable/options/#multiple-options), when `multiple=True`, then default value must be a list or a tuple. I'd set `default=()`.